src/python/pants/rules/core/BUILD

```diff
@@ -1 +1,17 @@
 python_library()
+
+python_tests(
+  name = 'roots',
```

Comment:
We want just one target for all tests in this dir, so name this `tests`, remove the `sources =` stanza (it'll default to all files with a `test_` prefix or ` _test` suffix, plus pytest's `conftest.py`).